src/components/MoviesList/MoviesList.tsx

```diff
@@ -5,12 +5,12 @@ import { MovieCard } from '../MovieCard';
 import { Movie } from '../../types/Movie';
 
 type Props = {
-  movies: Movie[];
+  movies: Movie[] | null;
```

Comment:
You don't need to render this component if there is no movies in the list 
```suggestion
  movies: Movie[];
```

Reply:
Thanks, done.